src/app/features/cases/components/organisms/CaseTimeline/TimelineCaseEvent.tsx

```diff
@@ -195,7 +196,7 @@ const ButtonWrap = styled.div`
 `
 
 const CaseEvent: React.FC<Props> = ({ caseEvents, button }) => {
-  const TimelineThread = caseEvents.map(thread =>
+  const TimelineThread = caseEvents?.map(thread =>
```

Comment:
`caseEvents` can not be undefined, right?

Reply:
I removed it